dev/src/components/__tests__/isValid.test.tsx

```diff
@@ -0,0 +1,54 @@
+import {isValid} from "../../Helpers"
+import {User, EnrollmentLink, Enrollment, Submission, SubmissionLink} from "../../../proto/ag/ag_pb";
+
+describe("User and enrollment validation", ()=> {
+    it("User should be valid", () =>{
+        const user = new User().setId(1).setName("Test User").setEmail("mail@mail.com").setStudentid("1234567")
+        const isValidUser = isValid(user)
+        expect(isValidUser).toBe(true)
+
+    });
+    it("User should not be valid if name is empty", () =>{
```

Comment:
This seems like a good place to insert a blank line. (So that each `it()` call is separate by blank lines)